content/en/data_jobs/databricks.md

```diff
@@ -57,29 +61,41 @@ You can choose to install the Agent globally, or on a specific Databricks cluste
+
+   In the **Environment variables** textbox, provide the values for the init script parameters.
+1. Click **Confirm**.
+
 {{% /tab %}}
 
 {{< /tabs >}}
 
-### Add your Datadog API key in Databricks
-1. Find your [Datadog API key][3].
-1. In Databricks, on the cluster configuration page, click the **Advanced options** toggle.
-1. At the bottom of the page, go to the **Spark** tab.
-   {{< img src="data_jobs/databricks/configure-databricks-spark-envvars-updated.png" alt="Databricks UI, cluster configuration advanced options, Spark tab. A textbox titled 'Environment variables' contains values for DD_API_KEY and DD_SITE." style="width:100%;" >}}
+### Init script parameters
 
-   In the **Environment variables** textbox, set values for: 
-    - `DD_API_KEY`: Your Datadog API key. (You can also use Databricks [secret management][4] to store your API key.)
-    - `DD_SITE`: Your [Datadog site][5].
+Provide the following parameters to the init script:
 
-   For example, if your Datadog site is {{< region-param key="dd_site" code="true" >}}, paste the following into the box:
+```text
+export DD_API_KEY=<YOUR API KEY>
+export DD_SITE={{< region-param key="dd_site" code="true" >}}
+export DATABRICKS_WORKSPACE=<YOUR WORKSPACE NAME>
+```
 
-   ```text
-   DD_API_KEY=<YOUR API KEY>
-   DD_SITE={{< region-param key="dd_site" code="true" >}}
-   ```
+Optionally, you can also set other init script parameters and Datadog environment variables here, such as `DD_ENV` and `DD_SERVICE`. The script can be configured using the following parameters:
+
+| Variable                 | Description                                                                                                                                                      | Default |
+|--------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| DD_API_KEY               | Your [Datadog API key][3]                                                                                                                                        |         |
+| DD_SITE                  | Your [Datadog site][5]                                                                                                                                           |         |
+| DATABRICKS_WORKSPACE     | Name of your Databricks Workspace. It should match the name provided in the [Datadog-Databricks integration step](#configure-the-datadog-databricks-integration) |         |
+| DD_PROCESS_AGENT_ENABLED | To enable monitoring of processes using Datadog's process agent                                                                                                  | false   |
```

Comment:
Let's remove `DD_PROCESS_AGENT_ENABLED` and `DD_PROFILING_ENABLED` from this list for now. I don't want to make it look like this product supports these yet (why not list other agent/tracer settings/config etc.) and there might be pricing/packaging around these capabilities in the future.